In [1]:
from math import sqrt

In [3]:
p1 = 164 / 15550
p2 = 228 / 15550
n1 = 164
n2 = 228
N1 = 15550
N2 = N1
p =  (n1 * p1 + n2 * p2) / (n1 + n2)
Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))
Z

-3.2110930329358367

Критическое значение Z-критерия при α=5% и двухсторонней альтернативе равно 1.96. Результат -3.2110930329358367<1.96. Выходит, изменения внесенные в заказную форму дали статистически значимый эффект на уровень конверсии в покупку. Рекомендуется использовать новую форму заказа.

При большом размере выборки, может быть использован z-критерий. 